cdap-common/src/main/java/co/cask/cdap/common/conf/Configuration.java

```diff
@@ -540,6 +540,16 @@ public String getRaw(String name) {
   }
 
   /**
+   * Get all key-value kafka config pairs
+   * @return hashmap of <code>name</code> property and its value
+   */
+  public Map<String, String> getKafkaConfigs() {
+    //use getValByRegex to get all config keys that start with "kafka."
+    String kafkaRegex = "^(kafka.*)$";
```

Comment:
so are you saying we change the way they are currently named? Because I don't believe that any of the configs have a `kafka.server.` prefix.